lib/commands/serve.js

```diff
@@ -72,7 +72,7 @@ module.exports = Command.extend({
           project: this.project
         });
 
-        return win.checkWindowsElevation(this.ui).then(function() {
+        return Win.checkIfSymlinksNeedToBeEnabled(this.ui).then(function() {
```

Comment:
Is this a static method somehow? It looks from the refactor that it's only on the prototype but this doesn't create an instance. Am I missing something?